keyvault/2016-10-01/swagger/keyvault.json

```diff
@@ -3049,6 +3051,11 @@
           "type": "string",
           "readOnly": true,
           "description": "The error message."
+        },
+        "innererror": {
```

Comment:
- This change is not supported in the previous api-version.  Callers of the previous api-version will continue to see the existing error format.
- This is the only change that is currently in this api-version.  The reason we are publishing a new api-version for this is that some of the older Key Vault client SDKs are known to throw exceptions when trying to deserialize objects containing unknown members (and we don't want to break existing applications).